src/main/java/com/karankumar/bookproject/ui/shelf/CustomShelfForm.java

```diff
@@ -98,8 +98,9 @@ private boolean customShelfNameAlreadyUsed(String customShelfName) {
     }
 
     private boolean customShelfNameMatchesPredefinedShelfName(String shelfName) {
+        new PredefinedShelfUtils(predefinedShelfService);
```

Comment:
Sorry, didn't get why we need to create an instance here and not using it anywhere?